snowpack/src/commands/dev.ts

```diff
@@ -1167,7 +1167,11 @@ export async function startDevServer(commandOptions: CommandOptions): Promise<Sn
         await result.checkStale();
       }
       if (result.contents) {
-        knownETags.set(reqPath, etag(result.contents, {weak: true}));
+        const tag = etag(result.contents, {weak: true});
+        knownETags.set(reqPath, tag);
+        if (reqPath.indexOf('?') >= 0) {
+          knownETags.set(reqPath.split('?', 2)[0], tag);
```

Comment:
This seems fine to me, but @FredKSchott would know more if this affects anything for the etag.

Reply:
Ah yea, good catch. We have another way of doing this in the rest of the codebase: `  const reqPath = decodeURI(url.parse(reqUrl).pathname!);` I'll update this PR to use that method.